interpreter/cling/lib/Interpreter/LookupHelper.cpp

```diff
@@ -1452,9 +1452,9 @@ namespace cling {
 
     DigestArgsInput inputEval;
     llvm::SmallVector<Expr*, 4> GivenArgs;
-    Interpreter::PushTransactionRAII pushedT(Interp);
```

Comment:
Could you add a comment in the commit why these are not needed anymore? Maybe I overlooked it?